tests/testdata_runrms/runrms.yml

```diff
@@ -17,49 +17,49 @@ rms:
   latest:
     exe: /prog/roxar/site/RMS_test_latest/rms/rms
     pythonpath:
-      - /prog/res/roxapi/x86_64_RH_7/matrix/12.1/lib/python3.8/site-packages
+      - /prog/res/roxapi/<PLATFORM>/matrix/12.1/lib/python3.8/site-packages
     pythonpathtest:
-      - /prog/res/roxapi/x86_64_RH_7/matrix/12.1_test/lib/python3.8/site-packages
+      - /prog/res/roxapi/<PLATFORM>/matrix/12.1_test/lib/python3.8/site-packages
     tcltkpath: /prog/roxar/site/RMS_test_latest/rms/versions
 
 
   11.1.2:
     default: true
     exe: rms -v 11.1.2
     pythonpath:
-      - /prog/res/roxapi/x86_64_RH_7/matrix/11.1/lib/python3.6/site-packages
+      - /prog/res/roxapi/<PLATFORM>/matrix/11.1/lib/python3.6/site-packages
     pythonpathtest:
-      - /prog/res/roxapi/x86_64_RH_7/matrix/11.1_test/lib/python3.6/site-packages
+      - /prog/res/roxapi/<PLATFORM>/matrix/11.1_test/lib/python3.6/site-packages
     pluginspath:
-      - /prog/res/roxapi/x86_64_RH_7/matrix/11.1/plugins
+      - /prog/res/roxapi/<PLATFORM>/matrix/11.1/plugins
     tcltkpath: /prog/roxar/rms/versions/11.1.2/lib/LINUX_64/tcl8.6
 
   12.0.1:
     exe: rms -v 12.0.1
     pythonpath:
-      - /prog/res/roxapi/x86_64_RH_7/matrix/12.0/lib/python3.6/site-packages
-      - /prog/res/roxapi/x86_64_RH_7/matrix/12.0/lib/python3.6/havana
+      - /prog/res/roxapi/<PLATFORM>/matrix/12.0/lib/python3.6/site-packages
+      - /prog/res/roxapi/<PLATFORM>/matrix/12.0/lib/python3.6/havana
     pythonpathtest:
-      - /prog/res/roxapi/x86_64_RH_7/matrix/12.0_test/lib/python3.6/site-packages
+      - /prog/res/roxapi/<PLATFORM>/matrix/12.0_test/lib/python3.6/site-packages
     pluginspath:
-      - /prog/res/roxapi/x86_64_RH_7/matrix/12.0/plugins
+      - /prog/res/roxapi/<PLATFORM>/matrix/12.0/plugins
     tcltkpath: /prog/roxar/rms/versions/12.0.1/lib/LINUX_64/tcl8.6
 
   12.0.2:
     exe: rms -v 12.0.2
     pythonpath:
-      - /prog/res/roxapi/x86_64_RH_7/matrix/12.0/lib/python3.6/site-packages
-      - /prog/res/roxapi/x86_64_RH_7/matrix/12.0/lib/python3.6/havana
+      - /prog/res/roxapi/<PLATFORM>/matrix/12.0/lib/python3.6/site-packages
+      - /prog/res/roxapi/<PLATFORM>/matrix/12.0/lib/python3.6/havana
     pythonpathtest:
-      - /prog/res/roxapi/x86_64_RH_7/matrix/12.0_test/lib/python3.6/site-packages
+      - /prog/res/roxapi/<PLATFORM>/matrix/12.0_test/lib/python3.6/site-packages
     pluginspath:
-      - /prog/res/roxapi/x86_64_RH_7/matrix/12.0/plugins
+      - /prog/res/roxapi/<PLATFORM>/matrix/12.0/plugins
     tcltkpath: /prog/roxar/rms/versions/12.0.1/lib/LINUX_64/tcl8.6
 
 # list of nonstandard installs (experimental)
 rms_nonstandard:
   12.0.0:
     exe: /prog/roxar/site/RMS12/rms/rms
     pythonpath:
-      - /prog/res/roxapi/x86_64_RH_7/matrix/12.0/lib/python3.6/site-packages
+      - /prog/res/roxapi/<PLATFORM>/matrix/12.0/lib/python3.6/site-packages
     tcltkpath: /prog/roxar/site/RMS12/rms/versions/12.0.0/lib/LINUX_64/tcl8.6
```

Comment:
These changes should be reverted @larsevj ?
You mentioned no rhel8 rms specific version available, and the current rhel7 version seems to be ok?